### DISCUSSION TITLE
Helm chart

### DISCUSSION THREAD

**Discussion by bmgante** [prymitive/karma#5601] [Q&A]
Hi everyone,
Is there some helm chart for karma? Would like to give it a try on a eks cluster.

Thanks

**Comment by LeoQuote** [comment]
next time you can search it on google or artifacthub 

https://artifacthub.io/packages/search?ts_query_web=karma&sort=relevance&page=1 

https://artifacthub.io/packages/helm/wiremind/karma

**Reply by luolong** [reply]
This is nice, but bearing in mind that this is probably going to be a recurring question, perhaps adding this to README installation instructions section, would nicely pre-empt further questions of the kind.

And just a bit of advice -- the tone of your response might have given off a bit of a hostile vibe. You might want to watch for that...

**Reply by LeoQuote** [reply]
Thanks for your advice on both technique and communication skills. But I just can’t help showing the hostile vibe on the question that clearly not searched by themselves, I think it’s also a kind of disrespect, but yeah maybe you’re right I should just stay silent and leave the question unanswered.
